src/KinDynWrapper/include/WalkingControllers/KinDynWrapper/Wrapper.h

```diff
@@ -77,6 +86,14 @@ namespace WalkingControllers
          */
         bool setBaseFrames(const std::string& lFootFrame, const std::string& rFootFrame);
 
+        /**
+         * Set The base frames for enabling using Gazebo as a base estimator.
+         * @param baseFrame the frame name inside model that supposed to be a root frame;
+         * @param name name of the frame attached to the base;
+         * @return true/false in case of success/failure.
+         */
+        bool setBaseFrame(const std::string& baseFrame, const std::string& name);
```

Comment:
>        This is probably just a documentation issue: 

Yes and we modified it as following:

```
  * @param baseFrame the frame name inside model;
  * @param name label for storing the frame information;
```

Reply:
> Yes and we modified it as following:

I don't understand. I still read:
```c++
	 * Set The base frames for enabling using Gazebo as a base estimator.
         * @param baseFrame the frame name inside model that supposed to be a root frame;
         * @param name name of the frame attached to the base;
         * @return true/false in case of success/failure.
```

Reply:
I hadn't pushed because I would liked to know your idea. Now I've pushed.